app/models/user.rb

```diff
@@ -2,10 +2,14 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: 'author_id'
   has_many :comments, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
-  validates :name, presence: true
-  validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
+  # validates :name, presence: true
+  # validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
- [ ] I think what the previous reviewer wanted you to do is to remove the commented-out code. Since it doesn't look very clean.